.golangci.yml

```diff
@@ -71,7 +71,7 @@ linters:
     - errcheck
     - errname
     # - errorlint
-    # - exhaustive
+    - exhaustive
```

Comment:
@hariso What made you realize we needed to enable this back?

Reply:
In one part of the code, there was a switch statement that was transforming proto statuses into Go statuses.